sdks/python/tox.ini

```diff
@@ -33,7 +33,7 @@ pip_pre = True
 # allow apps that support color to use it.
 passenv=TERM,CLOUDSDK_CONFIG,DOCKER_*,TESTCONTAINERS_*,TC_*,ALLOYDB_PASSWORD
 # Set [] options for pip installation of apache-beam tarball.
-extras = test,dataframe
```

Comment:
do we need to add yaml here: https://github.com/apache/beam/blob/40e2b0d01f26c20f792897638600bf162905252c/sdks/python/container/run_generate_requirements.sh#L68 ?